cockpit/IPC/src/views/Connections.tsx

```diff
@@ -29,7 +29,7 @@ const Connections:React.FC<DarkModeType> = ({ isDark }) => {
   return (
     <>
       <Title headingLevel="h1" size="2xl" style={{ color: isDark ? '#EEE' : '#111' }}>
-        IPC - Time For Change
+        Connections
```

Comment:
Erum við alveg á þessu?

Reply:
Felt unnecessarily long, and caused problems on mobile.
Could make font smaller, but It looks odd.

Suggestions?